tests/61push/08_rejected_pushers.pl

```diff
@@ -23,17 +23,15 @@ sub wait_for_push
    my ( $path, $response ) = @_;
 
    await_http_request( $path, sub {
-     my ( $request ) = @_;
-     my $body = $request->body_from_json;
-
-     return unless $body->{notification}{type};
-     return unless $body->{notification}{type} eq "m.room.message";
-     return 1;
-   })->then( sub {
       my ( $request ) = @_;
+      my $body = $request->body_from_json;
 
+      # Respond to all requests, even if we filiter them out
```

Comment:
```suggestion
      # Respond to all requests, even if we filter them out
```

Reply:
Where has my spell checker gone :/

Reply:
apparently he lives in Cambridge :angry: